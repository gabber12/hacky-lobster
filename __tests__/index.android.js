import 'react-native';
import React from 'react';
import App from '../index.android.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  fetch = jest.fn(() => new Promise(resolve => resolve()));
  const tree = renderer.create(
    <App />
  );
  expect(tree).toMatchSnapshot();

});
