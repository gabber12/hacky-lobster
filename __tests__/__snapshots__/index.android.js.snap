exports[`test renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      undefined,
    ]
  }>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "overflow": "hidden",
      }
    }>
    <View
      onStartShouldSetResponderCapture={[Function]}
      pointerEvents="auto"
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "overflow": "hidden",
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
              Object {
                "scaleX": 1,
              },
              Object {
                "scaleY": 1,
              },
              Object {
                "rotate": "0deg",
              },
              Object {
                "skewX": "0deg",
              },
              Object {
                "skewY": "0deg",
              },
            ],
          },
          null,
        ]
      }>
      <View
        style={
          Array [
            Object {
              "alignSelf": "flex-start",
              "borderColor": "red",
              "borderWidth": 1,
            },
            Object {
              "flex": 1,
            },
          ]
        }>
        <View
          style={
            Array [
              Object {
                "alignSelf": "flex-start",
                "borderColor": "red",
                "borderWidth": 1,
              },
              Object {
                "backgroundColor": "#FFF",
                "height": 50,
              },
            ]
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 20,
              }
            }>
            Hacker News
          </Text>
        </View>
        <ScrollView
          dataSource={
            ListViewDataSource {
              "items": "unknown",
            }
          }
          initialListSize={10}
          onContentSizeChange={[Function]}
          onEndReached={[Function]}
          onEndReachedThreshold={50}
          onKeyboardDidHide={undefined}
          onKeyboardDidShow={undefined}
          onKeyboardWillHide={undefined}
          onKeyboardWillShow={undefined}
          onLayout={[Function]}
          onScroll={[Function]}
          pageSize={1}
          refreshControl={
            <RefreshControl
              onRefresh={[Function]}
              refreshing={true} />
          }
          removeClippedSubviews={true}
          renderRow={[Function]}
          scrollEventThrottle={50}
          scrollRenderAheadDistance={1000}
          stickyHeaderIndices={Array []} />
      </View>
    </View>
  </View>
</View>
`;
